keps/sig-network/20200103-mixed-protocol-lb/README.md

```diff
@@ -487,6 +487,35 @@ From CPI implementation perspective thet feature can be graduated to beta, as th
 
 Graduating to GA means, that the feature flag checking is removed from the code. It means, that all CPI implementations must be ready to deal with Services with mixed protocol configuration - either rejecting such Services properly or managing the cloud load balancers according to the Service definition.
 
+#### Alpha -> Beta Graduation
```

Comment:
Since this KEP is delivering in alpha, add Alpha graduation criteria

Reply:
Added.